app/front-end/src/features/editor/components/toolbarView/toolbarGroup.tsx

```diff
@@ -0,0 +1,59 @@
+import { SvgIconComponent } from '@mui/icons-material';
+import { Box, Button, List, useTheme } from '@mui/material';
+
+export interface ToolbarGroupProps {
+  children: React.ReactNode;
+}
+
+export const ToolbarGroup: React.FC<ToolbarGroupProps> = ({ children }) => {
+  const Theme = useTheme();
+
+  return (
+    <List
+      sx={{
+        height: '75%',
+        bgcolor: Theme.palette.background.paper,
+        px: '1rem',
+        display: 'flex',
+        flexDirection: 'row',
+        flexWrap: 'wrap',
+        gap: '1rem',
+        overflow: 'auto',
+        alignContent: 'flex-start',
+      }}
+    >
+      {children}
+    </List>
+  );
+};
+
+export interface ToolbarGroupItemProps {
+  group: string;
+  icon: SvgIconComponent;
+  label: string;
+  onClick: () => void;
+}
```

Comment:
Interface ToolbarGroupitemProps and ToolbarGroup could be seperated into seperate files

Reply:
Seperated `jsx` components into different files. Interfaces are kept as close as possible to the `jsx` component.